docs/recipes/reasonml.md

```diff
@@ -0,0 +1,43 @@
+# [ReasonML](https://reasonml.github.io) / [BuckleScript](https://bucklescript.github.io)
+
+## Installation
+
+```sh
+npm install --save-dev ava
+```
+
+Then add `ava` to `bs-dev-dependencies` in your `bsconfig.json`:
+```js
+{
+  ...
+  "bs-dev-dependencies": ["ava"]
+}
+```
+
+## Getting started
+
+```ml
+open Sync;
+
+test("Should pass", t => {
+  t.deepEqual(1, 1);
+  t.pass();
+});
+```
+
+```ml
+open Async;
+
+test("Should pass", t => {
+  Js.Global.setTimeout(() => t.cb(), 1000)
```

Comment:
This should be `test.cb()` with a `t.end()` call in the timeout.

Reply:
`end` is a reserved word in ocaml. I try to find something similar. I'm open to discussion.